rsc-server/src/org/openrsc/model/GameMode.java

```diff
@@ -0,0 +1,43 @@
+package org.openrsc.model;
+
+/**
+ */
+public enum GameMode {
+
+	REGULAR(0),
+	IRONMAN(1),
+	HARDCORE_IRONMAN(2),
+	ULTIMATE_IRONMAN(3);

```

Comment:
Look up iron and game modes for runescape. I want to add them to the project, even if they're disabled, because it's something people will definitely want (as hosts and players)